cpp/include/cudf/filling.hpp

```diff
@@ -203,5 +203,23 @@ std::unique_ptr<column> sequence(
   scalar const& init,
   rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
 
+/**
+ * @brief Generate a sequence of timestamps of `n` rows, increment by calendrical months
+ *
+ * output[i] = `init` + i * `months`
+ *
+ * @param init The initial timestamp
+ * @param n Number of timestamps to generate
```

Comment:
```suggestion
 * @param size Number of timestamps to generate
```
Change references to `n` to `size`, right?
Also, the order of the parameters here do not match the signature.
And the `@param mr` is missing.